styles.css

```diff
@@ -0,0 +1,585 @@
+
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+.cw,
+.cw * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  background-color: white;
+}
+
+#header a {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
+
+.session-container {
+  display: none;
+}
+
+.secondary-nav {
+  display: none;
+}
+
+#menu {
+  position: absolute;
+  z-index: 1;
+}
+
+.close-menu-icon {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+}
+
+#menu-container {
+  position: absolute;
+  font-weight: bold;
+  top: 4em !important;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
- [ ] I see that you have used multiple times the `!important` keyword over your stylesheets, please do not get used to this as it bypasses any rules and there are better ways of applying styles such as using higher specificity selectors! 🤓 🔧

Reply:
Thank you for the suggestions @anagudelogu, I understand it's not such a good practice but I used it mainly for the more-less button functionality. :) I have removed that !important from line 67 as it is no longer of use. 👍

Reply:
- [ ] This issue is not well resolved as other parts of the styles has not been addressed :heavy_multiplication_x:

Reply:
Thank you I have removed other !important, properties that are not necessary :)

Reply:
- [ ] This issue is resolved.